client/state/selectors/get-current-user-payment-methods.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import { kebabCase, upperCase } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getCurrentUserLocale } from 'state/current-user/selectors';
+import { getGeoCountryShort } from 'state/geo/selectors';
+
+/**
+ * Constants
+ *
+ * Payment methods can be targeted via a WP.com locale code (actually
+ * a two letter lang usually) or by a full lang-CC locale.
+ */
+const default_payment_methods = [ 'credit-card', 'paypal' ];
+
+const paymentMethodsByLocale = {
```

Comment:
If we're planning to have this backed by an endpoint, it'd make sense to move this to a reducer that has this as initialState.